### PR TITLE
Quire docs/newsletter archive edit jp

### PR DESCRIPTION
**Pull request by pancaketents**
Newsletter archive January to May 2021

### CODE REVIEW

**Review by Erin-Cecele — CHANGES_REQUESTED** [review]
@pancaketents This looks great. "Community Forum in Action" repeats every month from March onwards. Feels a little redundant. Let's remove that as a headline. 

I'm also not sure if there's a way to add headlines for June and July because of the Javascript. @geealbers Is there a solution for that?

**Comment by geealbers** [comment]
No, you're right @Erin-Cecele, there's no way to do it since those links are being created through a script. Sorry. Might be an argument for adding the links manually each month and ditching the script? 😬

**Comment by Erin-Cecele** [comment]
Ok @geealbers I can try it out and see if it's cumbersome. @pancaketents on our am call let's chat about adding June and July manually. Thanks for all your work on this today!

File: content/community/newsletter-archive.md [hunk 1]
```diff
@@ -9,6 +9,8 @@ online: false
 
 <script type="text/javascript" src="https://newsletters.getty.edu/t/t/p/hulye/0/1/0/0/0/"></script>
```
Comment (Erin-Cecele) [inline content/community/newsletter-archive.md:10]: @pancaketents please remove this Javascript

**Review by Erin-Cecele — APPROVED** [review]

**Pull request merged**